src/app/modules/entity/entity-empty/entity-empty.component.ts

```diff
@@ -17,7 +18,7 @@ export class EntityEmptyComponent {
   }
 
   isLoading(): boolean {
-    return this.conf.type === EmptyType.Loading;
+    return this.conf.type === EmptyType.Loading && this.showSpinner;
```

Comment:
I'd rather we not show entity empty at all there rather than adding an obscure parameter.